### ISSUE TITLE
How to apply R3M to downstream robotic task?

### ISSUE THREAD

**Issue by yquantao** [facebookresearch/r3m#5]
Hi,

in the paper, it was mentioned that when we apply R3M to downstream robot learning tasks, only a few demonstrations are needed. My question is that do we also need to add language in these demos? Or only video demos are needed?

**Comment by suraj-nair-1** [comment]
No language is needed (only used during training), the pre-trained R3M model simply acts as an encoder mapping images to embeddings. So to train with imitation learning you just need data of (Image, action) pairs, and you can encode the images with R3M and train with your normal imitation learning loss. You can see an example of encoding a single image [here](https://github.com/facebookresearch/r3m/blob/main/r3m/example.py)

Best,
Suraj

**Comment by whc688** [comment]
@suraj-nair-1 Hi! 
If i want to use frozen pre-trained R3M as an encoder to encode image, is there any kind of image normalization i need to apply on the image? and what should i apply on it?
Thanks a lot!
